cmd/elasticindexer/flags.go

```diff
@@ -35,4 +35,10 @@ var (
 		Name:  "disable-ansi-color",
 		Usage: "Boolean option for disabling ANSI colors in the logging system.",
 	}
+	importDB = cli.BoolFlag{
```

Comment:
wondering if this can be removed in the future as the node can send any other operation modes or settings. Reason: we can easily forget to set the -import-mode on the indexer resulting in some inconsistencies that require time to investigate